pymoveit2/moveit2.py

```diff
@@ -872,6 +894,52 @@ def _plan_kinematic_path(
                 f"Planning failed! Error code: {res.error_code.val}."
             )
             return None
+    def _plan_cartesian_path(
+        self, wait_for_server_timeout_sec: Optional[float] = 1.0
+    ) -> Optional[JointTrajectory]:
+
+        # Re-use request from move action goal
+        self.__cartesian_path_request.start_state = self.__move_action_goal.request.start_state
+        self.__cartesian_path_request.group_name = self.__move_action_goal.request.group_name
+        self.__cartesian_path_request.link_name = self.__end_effector_name
+        self.__cartesian_path_request.max_step = 0.0025
```

Comment:
It would be nice to have `cartesian_max_step` parameter (or something similar) exposed to the high-level interface. It is okay if not, it can be added later if someone needs it - together with `jump_threshold` and others.
```suggestion
    def _plan_cartesian_path(
        self, 
        max_step: float = 0.0025,
        wait_for_server_timeout_sec: Optional[float] = 1.0,
    ) -> Optional[JointTrajectory]:

        # Re-use request from move action goal
        self.__cartesian_path_request.start_state = self.__move_action_goal.request.start_state
        self.__cartesian_path_request.group_name = self.__move_action_goal.request.group_name
        self.__cartesian_path_request.link_name = self.__end_effector_name
        self.__cartesian_path_request.max_step = max_step
```